diff --git a/BUILD.gn b/BUILD.gn
index df31ad6..b9386f9 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -21,10 +21,7 @@ import("third_party/dawn/args.gni")
 config("skia_public") {
   include_dirs = [ "." ]
 
-  defines = [
-    "SK_CODEC_DECODES_BMP",
-    "SK_CODEC_DECODES_WBMP",
-  ]
+  defines = []
   cflags_objcc = []
   if (is_component_build) {
     defines += [ "SKIA_DLL" ]
@@ -1302,7 +1299,6 @@ optional("png_decode_common") {
 optional("png_decode_libpng") {
   enabled = skia_use_libpng_decode
   public_defines = [
-    "SK_CODEC_DECODES_ICO",
     "SK_CODEC_DECODES_PNG",
     "SK_CODEC_DECODES_PNG_WITH_LIBPNG",
   ]
@@ -1312,7 +1308,7 @@ optional("png_decode_libpng") {
     ":png_decode_common",
     "//third_party/libpng",
   ]
-  sources = [ "src/codec/SkIcoCodec.cpp" ] + skia_codec_libpng_srcs
+  sources = skia_codec_libpng_srcs
 
   # Export this so fiddles could decode png using libpng
   public = skia_codec_libpng_public
@@ -2040,7 +2036,6 @@ skia_component("skia") {
   sources += skia_colorfilters_sources
   sources += skia_effects_imagefilter_sources
   sources += skia_codec_shared
-  sources += skia_codec_decode_bmp
   sources += skia_encode_srcs
   sources += skia_sksl_core_sources
   sources += skia_sksl_core_module_sources
